controllers/cloud.redhat.com/providers/iqe/impl.go

```diff
@@ -244,7 +244,8 @@ func CreateIqeJobResource(cache *rc.ObjectCache, cji *crd.ClowdJobInvocation, en
 		"ignore-check.kube-linter.io/no-liveness-probe":  "probes not required on Job pods",
 		"ignore-check.kube-linter.io/no-readiness-probe": "probes not required on Job pods",
 	}
-	utils.UpdatePodTemplateAnnotations(&j.Spec.Template, annotations)
+	utils.UpdateAnnotations(&j.Spec.Template, annotations)
+	utils.UpdateAnnotations(&j.ObjectMeta, annotations)
```

Comment:
Again here I'd update the job itself, not the ObjectMeta, but does the job itself need these annotations?